### PR TITLE
spread gun works now with drop gun

### PR DESCRIPTION
**Pull request by fokkonaut**

### CODE REVIEW

File: src/game/server/entities/character.cpp [hunk 1]
```diff
@@ -5015,17 +5015,48 @@ void CCharacter::DropWeapon(int WeaponID)
...
 	{
-		m_Jetpack = false;
-		GameServer()->SendChatTarget(GetPlayer()->GetCID(), "You lost your jetpack gun");
+		if (m_Jetpack && (m_autospreadgun || m_pPlayer->m_InfAutoSpreadGun))
+		{
+			m_Jetpack = false;
+			m_autospreadgun = false;
+			m_pPlayer->m_InfAutoSpreadGun = false;
+			GameServer()->SendChatTarget(GetPlayer()->GetCID(), "You lost your jetpack gun");
+			GameServer()->SendChatTarget(GetPlayer()->GetCID(), "You lost your spread gun");
```
Comment (ChillerDragon) [inline src/game/server/entities/character.cpp]: Two different messages at the same time?

Comment (fokkonaut) [reply src/game/server/entities/character.cpp]: whats the problem?

Comment (ChillerDragon) [reply src/game/server/entities/character.cpp]: sowwy me nobo

Comment (fokkonaut) [reply src/game/server/entities/character.cpp]: true

**Pull request merged**